api/process_bundles.go

```diff
@@ -55,6 +56,16 @@ func (a *API) createProcessBundleHandler(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	org, err := a.db.Organization(census.OrgAddress)
+	if err != nil {
+		if err == db.ErrNotFound {
+			errors.ErrMalformedURLParam.Withf("organization not found").Write(w)
```

Comment:
why not errors.ErrOrganizationNotFound.Write(w)?

(btw orgaddress is not a URL param in this endpoint)